src/models/Projection.ts

```diff
@@ -277,36 +305,69 @@ export class Projection {
     return result;
   }
 
-  private calcIcuUtilization(
+  private useActualTimeseries(actualTimeseries: ActualsTimeseries) {
+    // make sure there are recent datapoints.
+    if (actualTimeseries.length < 3) return false;
```

Comment:
Maybe move `3` and `5` into constants?
```
/** 
 * We look at the last 5 days of actuals and require at least 3 data points to 
 * be present in order to consider the data "fresh" enough to use. 
 */
const ACTUALS_FRESHNESS_WINDOW_DAYS = 5;
const ACTUALS_FRESHNESS_REQUIRED_DAYS = 3;
```

Reply:
this is total garbage and not what we agreed on i was just trying to view the charts >.<